### DISCUSSION TITLE
Adding code competion for web.config

### DISCUSSION THREAD

**Discussion by AJNijland** [VsixCommunity/Community.VisualStudio.Toolkit#276] [Q&A] [answered]
Hi all,
I'm looking for a way to add Code Completion for a web.config file. When a user adds an appSetting, I want to present the user a list of possible values from which he can choose. 

I took a glance at the great work of Mads with his PkgdefLanguage and MarkdownEditor, but it's 'creating' its own file extension/language and not extending the existing xml-config language. Couldn't find a way to do it yet.

Is it possible to extend the IntelliSense or Code Completion for .config files at all?

**Comment by madskristensen** [comment, marked as answer]
I do that in my VSCT IntelliSense extension but for .vsct files.

**Reply by AJNijland** [reply]
Oke, thanks! I will take a look at that one tomorrow!
